pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: google_fonts
 description: A package to include fonts from fonts.google.com in your flutter app.
-version: 0.5.0+1
+version: 0.6.0+1
```

Comment:
sync to 0.6.x

Reply:
Agreed, remove the `+1` from the version number here.